cli_tools/gce_ovf_import/ovf_import_params/ovf_import_params.go

```diff
@@ -49,7 +49,7 @@ type OVFImportParams struct {
 	BootDiskKmsLocation         string
 	BootDiskKmsProject          string
 	Timeout                     string
-	Project                     string
+	Project                     *string
```

Comment:
why the change to pointer?

Reply:
"project" is a input param. However, in many cases, user didn't type in this value (so it will be initialized as an empty value). Thus, the 'project' will be populated after the tool is lunched. We need that populated value. A pointer gives us the ability to reflect the updated 'project' in log.

Reply:
Can we explicitly update params to make this workflow obvious?

Reply:
"updateParam" is called not very deep, so it's realtively explicit from my perspective.
It's hard to put that in "main" function. The design tried to separate logging service from import/export logic, so it can't be that explicit.

Reply:
where is project updated after the fact?

Reply:
log_entry.go -> "updateParam". The reason why it's not clear in this PR is becuase the code is partially landed in the last PR which I only fixed image-export. Link here: https://github.com/GoogleCloudPlatform/compute-image-tools/pull/970